### PR TITLE
WAN-video: support training with batch size > 1

### PR DESCRIPTION
**Pull request by hnyu**
Currently, if we want to set batch size > 1 for finetuning WAN, there will be dim0 size mismatch at 
```python
x = torch.cat([x, y], dim=1)  # (b, c_x + c_y, f, h, w)
``` 
in `wan_video_dit.py`. The reason is that the code is hardcoded to assume batch_size=1 when processing "context", "clip_feature" and "y". We should actually use `squeeze(1)`.

### CODE REVIEW

**Comment by Artiprocher** [comment]
@hnyu Sorry for the late reply. I'm the main developer of this open-source project, and I just graduated not long ago, so I took an extended break. I've now returned to my regular work schedule.

We truly understand your request. However, we believe that the engineering effort required to support a batch size greater than 1 is not justified by the limited benefits it brings.

On one hand, most GPUs do not have enough memory to support training with a batch size greater than 1. On the other hand, using gradient accumulation can effectively achieve the same result.

Additionally, most developers of the Wan series models we've integrated did not implement support for batch sizes larger than 1, which would require us to conduct extensive testing to enable this feature. For these reasons, we currently have no plans to support batch sizes greater than 1.

We have updated our training and inference framework, and both multi-GPU training and inference are now supported. Feel free to try them out!

Thanks for your contribution. But we have to close this PR now because of our new training framework.
